### ISSUE TITLE
string-quotes in // comments

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#547]
```scss
// Automatically set modal's width for larger viewports
.test {
    color: #fff;
}
```

Error:
```bash
1:27	⚠  Expected double quotes (string-quotes) [stylelint]
```

I think we don't need checking string-quotes types in comments.

**Comment by davidtheclark** [comment]
This is specifically an issue for non-standard `//` comments.

Actually, this raises a big flag ... `styleSearch()` does not account for these `//` comments, so wherever it is used I'd expect the same issue.

**Comment by evilebottnawi** [comment]
@davidtheclark Hm. While proposing to get rid of the problem?

**Comment by davidtheclark** [comment]
> While proposing to get rid of the problem?

I don't understand the question.

**Comment by evilebottnawi** [comment]
@davidtheclark What can be done to ignored string-quotes in comments?

**Comment by davidtheclark** [comment]
We'll have to figure out some way of modifying the `styleSearch()` utility to address these comments.

**Comment by evilebottnawi** [comment]
@davidtheclark how i can help? :)

**Comment by davidtheclark** [comment]
Thanks — don’t worry about it for now. That `styleSearch()` util is kind of beastly. I will look into it tonight and see what options there are, and I’ll let you know if you can help then.

**Comment by evilebottnawi** [comment]
@davidtheclark thanks
